Sources/Core/AWSSDKIdentity/Tests/AWSSDKIdentityTests/AWSCredentialIdentityResolverTests/CachedAWSCredentialIdentityResolverTests.swift

```diff
@@ -9,13 +9,15 @@ import XCTest
 import struct AWSSDKIdentity.CachedAWSCredentialIdentityResolver
 
 class CachedAWSCredentialIdentityResolverTests: XCTestCase {
```

Comment:
In this and the tests below, accountID has been added to the test to ensure that it is resolved along with other AWSCredentialIdentity fields.